### PR TITLE
Minor fixes for validation.

### PR DESCRIPTION
**Pull request by seriva**
- Added fix for input document validation
- Automatic resolve for  yes|no to true|false

### CODE REVIEW

**Review by erzetpe — DISMISSED** [review]

**Review by atsikham — DISMISSED** [review]

File: core/src/epicli/cli/helpers/objdict_helpers.py [hunk 1]
```diff
@@ -52,3 +52,18 @@ def remove_value(d, value):
...
+def replace_yesno_with_booleans(d):
+    if isinstance(d, list):
+        for dd in d:
+            replace_yesno_with_booleans(dd)
+    elif isinstance(d, ObjDict):
+        for key, val in d.items():
+            if isinstance(d[key], str):
+                if val == 'yes':        
+                    d[key] = True
+                if val == 'no':
```
Comment (to-bar) [inline core/src/epicli/cli/helpers/objdict_helpers.py]:
```suggestion
                elif val == 'no':             
```

**Review by to-bar — CHANGES_REQUESTED** [review]

**Comment by przemyslavic** [comment]
❤️

File: core/src/epicli/cli/helpers/objdict_helpers.py (hunk 1 above)
Comment (seriva) [reply core/src/epicli/cli/helpers/objdict_helpers.py]: Fixed.

**Review by przemyslavic — APPROVED** [review]

**Review by to-bar — APPROVED** [review]

**Pull request merged**